dashboard/app/views/pd/workshop_mailer/organizer_cancel_receipt.html.haml

```diff
@@ -9,7 +9,7 @@
   has canceled their registration for your
   = @workshop.course
   = @workshop.subject
-  = link_to 'workshop', "/pd/workshop_dashboard/workshops/#{@workshop.id}"
+  = link_to 'workshop', CDO.studio_url("/pd/workshop_dashboard/workshops/#{@workshop.id}"), CDO.default_scheme
```

Comment:
The scheme should be an argument to [CDO.studio_url](https://github.com/code-dot-org/code-dot-org/blob/8dced41baceffb096ed16154bc585d3132713a61/deployment.rb#L234), not [link_to](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to).

Like this:
```ruby
link_to 'workshop', CDO.studio_url("/pd/workshop_dashboard/workshops/#{@workshop.id}", CDO.default_scheme)
```